namespace_conn.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/googollee/go-socket.io/parser"
 )
 
-// Namespace
+// Namespace is interface describe namespace functionality
```

Comment:
Not really sure what this means?

Should it be something like?

```go
// Namespace describes a socket.io room
```

or similar?

Reply:
Nice one. I changed